src/sql/base/browser/ui/panel/panel.ts

```diff
@@ -161,19 +164,42 @@ export class TabbedPanel extends Disposable {
 			invokeTabSelectedHandler();
 		}));
 
-		tab.disposables.push(DOM.addDisposableListener(tabHeaderElement, DOM.EventType.KEY_UP, (e: KeyboardEvent) => {
+		tab.disposables.push(DOM.addDisposableListener(tabHeaderElement, DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
```

Comment:
Changing to Key down because that's called before Key up. It was not a part of accessibility bug but issues occurs when 
1) a tab is filled with enough/default info (to click ok)
2) but you focus to next tab
3) press enter to open the next tab
it used to close the dialog rather than opening the new selected tab, which was wrong.

Let me know if there was some other reason to use key up